roles/security_group/requirements.txt

```diff
@@ -0,0 +1 @@
+openstacksdk>=0.12.0
```

Comment:
In order for tests to pass, this file needs to live in (or be sym linked from) `roles/security_group/molecule/*/requirements.txt`